app/components/lr-progress.js

```diff
@@ -0,0 +1,55 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  classNames: ['lr-progress'],
+
+  classNameBindings: ['isCompleted:lr-progress--is-completed', 'success:lr-progress--success'],
+
+  reached: 0,
+
+  total: 100,
+
+  withAchievementPoints: false,
+
+  percentage: Ember.computed('reached', 'total', function () {
+
+    return this.get('reached') / this.get('total') * 100;
+
+  }),
+
+  isCompleted: Ember.computed('percentage', function () {
+
+    if (Number(this.get('percentage')) === 100) {
+
+      return true;
+
+    } else {
+
+      return false;
+
+    }
+
+  }),
+
+  inProgress: Ember.computed('percentage', function () {
+
+    if (Number(this.get('percentage')) < 100 && Number(this.get('percentage')) > 0) {
```

Comment:
It's better to remove the second condition, because it's not neccessary. It should **always** be "inProgress" when it's below 100%